src/Identity/testassets/Identity.DefaultUI.WebSite/StartupBase.cs

```diff
@@ -42,10 +42,11 @@ public virtual void ConfigureServices(IServiceCollection services)
             services.AddDbContext<TContext>(options =>
                 options
                     .ConfigureWarnings(b => b.Log(CoreEventId.ManyServiceProvidersCreatedWarning))
-                    .UseSqlServer(
-                        Configuration.GetConnectionString("DefaultConnection"),
-                        sqlOptions => sqlOptions.MigrationsAssembly("Identity.DefaultUI.WebSite")
-                    ));
+                    //.UseSqlServer(
```

Comment:
Nit: cleanup?

Reply:
I left it commented on purpose so that its easy to test with SQL Server. I need to follow up with @ajcvickers on why SetupTestDatabase wasn't overriding it, which is what I was expecting to happen.